torch/testing/_internal/common_methods_invocations.py

```diff
@@ -10619,6 +10627,48 @@ def error_inputs_mean(op_info, device, **kwargs):
                                      'test_reference_numerics_extremal_values',
                                      dtypes=(torch.complex64, torch.complex128)),
                     )),
+    BinaryUfuncInfo('clamp_max',
+                    ref=_clamp_max_numpy,
+                    dtypes=all_types_and(torch.bfloat16),
+                    dtypesIfCUDA=all_types_and(torch.half, torch.bfloat16, torch.float16),
+                    supports_forward_ad=True,
+                    supports_rhs_python_scalar=False,
+                    supports_fwgrad_bwgrad=True,
+                    rhs_make_tensor_kwargs=dict(exclude_zero=False),
+                    skips=(
+                        # clamp_max supports two tensor input with bool, but not a bool scalar
+                        DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_dtypes'),
```

Comment:
Some sample inputs failing for dtype shouldn't result in test failure, so what's going on here?

Reply:
This test was complaining about missing `torch.bool` and `torch.float16` in dtypes. 
```
======================================================================
FAIL: test_dtypes_clamp_max_cpu (__main__.TestCommonCPU)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/pytorch/pytorch/torch/testing/_internal/common_device_type.py", line 377, in instantiated_test
    result = test(self, **param_kwargs)
  File "/opt/pytorch/pytorch/torch/testing/_internal/common_device_type.py", line 786, in test_wrapper
    return test(*args, **kwargs)
  File "/opt/pytorch/pytorch/torch/testing/_internal/common_device_type.py", line 821, in dep_fn
    return fn(slf, *args, **kwargs)
  File "/opt/pytorch/pytorch/torch/testing/_internal/common_device_type.py", line 979, in only_fn
    return fn(self, *args, **kwargs)
  File "test_ops.py", line 314, in test_dtypes
    self.fail(msg)
AssertionError: The supported dtypes for clamp_max on device type cpu are incorrect!
The following dtypes worked in forward but are not listed by the OpInfo: {torch.bool, torch.float16}.
The following dtypes worked in backward but are not listed by the OpInfo: {torch.float16}.
```
The comment here `# clamp_min supports two tensor input with bool, but not a bool scalar` was referring to the failure on a different test when I add `torch.bool` in the supported dtype. (I think I also mistakenly set `rhs_python_scalar=True` then).